DemoApp/Sources/Views/CallView/CallingView/DetailedCallingView.swift

```diff
@@ -140,12 +140,16 @@ struct DetailedCallingView: View {
             self.callId = callId
             viewModel.joinCall(callType: .default, callId: callId)
         }
+        .onReceive(appState.$deeplinkInfo) { deeplinkInfo in
```

Comment:
Added this one to align the DetailedCallingView with the SimpleView and allow us to perform the DeeplinkTests